libp2p/pubsub/gossipsub.py

```diff
@@ -603,6 +586,12 @@ def _get_in_topic_gossipsub_peers_from_minus(
         rpc_msg: bytes = packet.SerializeToString()
 
         # 3) Get the stream to this peer
+        if sender_peer_id not in self.pubsub.peers:
+            logger.debug(
+                "Fail to responed to iwant request from %s: peer disconnected",
```

Comment:
The wording "peer disconnected" might be misleading here. The symptom is we can't find the peer in `pubsub.peers` but the wording assumes the cause of it. Maybe just say `"can't find {sender_peer_id} in pubsub.peers"`?